content/microservices/redis.md

```diff
@@ -1,20 +1,20 @@
 ### Redis
 
-A second built-in transporter is based on [Redis](https://redis.io/) database. This transporter takes advantage of **publish/subscribe** feature.
+[Redis](https://redis.io/) transporter implements the publish/subscribe messaging paradigm and leverages [Pub/Sub](https://redis.io/topics/pubsub) feature of Redis. Published messages are categorised in channels, without knowing what subscribers will eventually receive the message (and if any). Each microservice can subscribe to any number of channels. In addition, more than one channel can be subscribed to at a time. Messages exchanged through channels are **fire-and-forget**, which means that if a message is published and there are no subscribers interested in it, the message removed and cannot be recovered. Thus, you don't have a guarantee that either messages or events will be handled by at least one service. Also, single message can be received many subscribers.
 
 <figure><img src="/assets/Redis_1.png" /></figure>
 
 #### Installation
 
-Before we start, we have to install required package:
+To start building Redis-based microservices, first install the required package:
 
 ```bash
 $ npm i --save redis
 ```
 
 #### Overview
 
-In order to switch from TCP transport strategy to Redis **pub/sub**, we need to change an options object passed to the `createMicroservice()` method.
+In order to use Redis transporter, a different options object has to be passed to the `createMicroservice()` method.
```

Comment:
To use the Redis transporter, pass the following options object to the `createMicroservice()` method: